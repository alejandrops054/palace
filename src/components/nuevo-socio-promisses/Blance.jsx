import React from 'react'

/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

const Blanco = {
    color: "white"
}

const Blance = () => {
    return (
        <div className="border-right border-secondary">
            <div>
                <h1>Blacen Financing</h1>
                <h1>$ <span style={Blanco}>0.00 </span> </h1>
            </div>
        </div>
    )
}

export default Blance;