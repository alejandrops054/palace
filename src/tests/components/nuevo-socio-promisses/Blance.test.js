/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import Blance from '../../../components/nuevo-socio-promisses/Blance'

describe('Blance component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<Blance/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})