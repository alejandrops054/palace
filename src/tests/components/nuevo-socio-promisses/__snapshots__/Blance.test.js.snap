// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blance component Debe mostrar snapshot 1`] = `
<div
  className="border-right border-secondary"
>
  <div>
    <h1>
      Blacen Financing
    </h1>
    <h1>
      $ 
      <span
        style={
          Object {
            "color": "white",
          }
        }
      >
        0.00 
      </span>
       
    </h1>
  </div>
</div>
`;
